.github/ISSUE_TEMPLATE/---feature-request.md

```diff
@@ -1,23 +1,23 @@
 ---
 name: "\U0001F381 Feature Request"
-about: Do you have ideas for new features and improvements?
+about: Do you have ideas for new features or improvements?
 title: ''
 labels: kind/feature, status/triage
 assignees: ''
 
 ---
 
 <!--
-  Hi there! Thank you for wanting to make Poetry better.
+    Hi there! Thank you for submitting a bug report!
 
-  Before you submit this; let's make sure of a few things.
-  Please make sure the following boxes are ticked if they are correct.
-  If not, please try and fulfill these first.
+    Before you submit your issue, please review and follow the instructions at
+    https://python-poetry.org/docs/contributing/#suggesting-enhancements.
 -->
 
-<!-- Checked checkbox should look like this: [x] -->
+<!-- All the below steps should be completed before submitting your issue. Checked checkbox should look like this: [x] -->
```

Comment:
```suggestion
<!-- All the below steps should be completed before submitting your request. Checked checkbox should look like this: [x] -->
```

Reply:
I like issue as a generic descriptor of anything on the issue tracker